.github/workflows/code-quality-pr.yaml

```diff
@@ -31,6 +37,6 @@ jobs:
         run: echo '${{ steps.file_changes.outputs.files}}'
 
       - name: Run pre-commits
-        uses: pre-commit/action@v3.0.0
+        uses: pre-commit/action@v3.0.1
         with:
-          extra_args: --files ${{ steps.file_changes.outputs.files}}
```

Comment:
For the PR workflow this is likely more appropriate. Why did you remove it?

Reply:
Ah ok, I referenced the MEDS-DEV workflow but yeah original one for PR makes sense